gcloud/logging/client.py

```diff
@@ -53,3 +54,22 @@ def logger(self, name):
         :returns: Logger created with the current client.
         """
         return Logger(name, client=self)
+
+    def sink(self, name, filter_, destination):
+        """Creates a sink bound to the current client.
```

Comment:
Per the [sink API docs](https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/projects.sinks), linked from the `Sink` class docstring:  sinks are named, filtered exports of log entries to externals stores (buckets, pubsub topics, or bigquery datasets.